q2_fondue/scraper.py

```diff
@@ -189,23 +372,25 @@ def scrape_collection(
             logger.warning(f'Item {attach_key} doesn\'t contain any '
                            f'full-text content')
 
-        # find run IDs
-        run_ids += _find_accession_ids(str_text, 'run')
-        # find bioproject IDs
-        bioproject_ids += _find_accession_ids(str_text, 'bioproject')
+        # find accession (run and bioproject) IDs
+        run_ids = _find_accession_ids(str_text, 'run')
+        bioproject_ids = _find_accession_ids(str_text, 'bioproject')
 
-    # remove duplicate entries in both lists
-    run_ids = list(set(run_ids))
-    bioproject_ids = list(set(bioproject_ids))
+        # match found accession IDs with DOI
+        run_doi = _merge2dict(run_doi, run_ids, doi)
+        bioproject_doi = _merge2dict(
+            bioproject_doi, bioproject_ids, doi)
 
-    if (len(run_ids) == 0) & (len(bioproject_ids) == 0):
+    if (len(run_doi) == 0) & (len(bioproject_doi) == 0):
         raise NoAccessionIDs(f'The provided collection {collection_name} '
                              f'does not contain any run or BioProject IDs')
-    elif len(run_ids) == 0:
+    elif len(run_doi) == 0:
         logger.warning(f'The provided collection {collection_name} '
                        f'does not contain any run IDs')
```

Comment:
I saw the logging level was set to `INFO`, but didn't see this in stdout for a collection with BioProject IDs but no run IDs. Is logging written to file by default? Since this is a warning, it could be worth writing to console for the user.

Reply:
Did you run the command with `--verbose`? As with `--verbose`, the INFO statements are printed to the console in my case.

Reply:
True! But since this one's a warning, I was wondering if it'd be helpful to print to console even without `--verbose`. Makes sense if you'd want to keep stdout clean by default, though.

Reply:
Hm, as far as I know there is no option of writing it to console without `--verbose` in QIIME2.